superset/assets/src/explore/components/controls/DateFilterControl.jsx

```diff
@@ -28,172 +51,265 @@ const propTypes = {
 const defaultProps = {
   animation: true,
   onChange: () => {},
-  value: '',
+  value: 'Last week',
 };
 
+
 export default class DateFilterControl extends React.Component {
   constructor(props) {
     super(props);
-    const value = props.value || '';
+    const value = props.value || defaultProps.value;
```

Comment:
isn't this redundant of how react already deals with defaultProps?

Reply:
Yeah, I was unsure about this and was afraid of breaking things. I assume it's protecting cases where `value` is explicitly set to `null`?

Reply:
PropTypes will prevent against null being passed in- I'd say drop the `|| defaultProps.value`.